builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -288,6 +292,13 @@ node() {
             lambdaEvents.updateLambdaPermissionAndNotification(lambdaARN, s3BucketName, config['event_action_s3'])
           }
 
+          if (config['event_source_dynamodb'] || config['event_source_sqs']) {
+            def event_source = (config['event_source_dynamodb']? 'event_source_dynamodb': 'event_source_sqs' ).split("_")[2]
+            def role_name =  "${configLoader.INSTANCE_PREFIX}-${config['domain']}-${config['service']}-${environment_logical_id}"
```

Comment:
Is there a better way to get the ARNs of the roles that get created? `sls --info` may be?

Reply:
Tried serverless info , list-stack-resouces, get-template etc. Not get the role arn. So going with the current approach. Will keep this and raise a follow up PR, for the same in future.